### PR TITLE
Feat: Render rich text

### PR DESCRIPTION
**Pull request by manuelschroederdev**
New function to render rich text more conveniently

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -190,6 +190,16 @@ sbBridge.on(["input", "published", "change"], (event) => {
 });
 ```
 
+#### Rendering Rich Text
+
+You can easily render rich text by using the `richTextResolver` function that comes with `@storyblok/js`:
```
Comment (fgiuliani) [inline README.md:195]:
```suggestion
You can easily render rich text by using the `renderRichText` function that comes with `@storyblok/js`:
```

File: lib/__tests__/__snapshots__/index.test.js.snap [hunk 2]
```diff
@@ -0,0 +1,14 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`@storyblok/js Rich Text Resolver should return and empty string and warn in console when it's a falsy value 1`] = `
```
Comment (fgiuliani) [inline lib/__tests__/__snapshots__/index.test.js.snap:3]:
```suggestion
exports[`@storyblok/js Rich Text Resolver should return an empty string and warn in console when it's a falsy value 1`] = `
```

File: lib/cypress/integration/index.spec.js [hunk 3]
```diff
@@ -1,3 +1,5 @@
+import { describe } from "mocha";
```
Comment (fgiuliani) [inline lib/cypress/integration/index.spec.js:1]: where is `describe` being used? 🤔

File: package.json [hunk 4]
```diff
@@ -58,5 +58,8 @@
     "tabWidth": 2,
     "semi": true,
     "singleQuote": false
+  },
+  "dependencies": {
+    "cypress": "^10.3.0"
```
Comment (fgiuliani) [inline package.json:63]: shouldn't this be a devDependencies?

File: lib/index.ts [hunk 5]
```diff
@@ -67,9 +72,20 @@ export const storyblokInit = (pluginOptions: SbSDKOptions = {}) => {
   return result;
 };
 
-export const loadStoryblokBridge = () => {
-  return loadBridge(bridgeLatest);
-};
+export const renderRichText = (text: Richtext) : string => {
+  if (text as any === '') {
```
Comment (fgiuliani) [inline lib/index.ts:76]: shouldn't this be "as string"?

File: lib/index.ts [hunk 6]
```diff
@@ -67,9 +72,20 @@ export const storyblokInit = (pluginOptions: SbSDKOptions = {}) => {
...
+  else if(!text) {
+    console.warn(`${text} is not a valid Richtext object. This might be because the value of the richtext field is empty.
+    
+  For more info about the richtext object check https://github.com/storyblok/storyblok-js#rendering-rich-text`)
+    return ""; 
+  }
+  return resolver.render(text)
+}
+
+export const loadStoryblokBridge = () => loadBridge(bridgeLatest)
```
Comment (fgiuliani) [inline lib/index.ts:88]: what would be the purpose/idea of this function?

File: package.json (hunk 4 above)
Comment (alexjoverm) [reply package.json:63]: Yes, great point

File: lib/index.ts (hunk 5 above)
Comment (alexjoverm) [reply lib/index.ts:76]: both would work actually

File: lib/cypress/integration/index.spec.js (hunk 3 above)
Comment (alexjoverm) [reply lib/cypress/integration/index.spec.js:1]: @manuelschroederdev ... your plugin is trying to be too funny

**Review by alexjoverm — CHANGES_REQUESTED** [review]
Let's polish these few changes 😉  Also @manuelschroederdev the build seems to be failing because of an import https://github.com/storyblok/storyblok-js/runs/7451908250?check_suite_focus=true

File: lib/cypress/integration/index.spec.js (hunk 3 above)
Comment (manuelschroederdev) [reply lib/cypress/integration/index.spec.js:1]: @manuelschroederdev I just checked, it's not even a plugin that's responsible for this. 👀

File: lib/index.ts (hunk 6 above)
Comment (manuelschroederdev) [reply lib/index.ts:88]: @fgiuliani That function was already added a couple of months ago when I started working with Astro. Alex, you and I realized that for some frameworks, there should be the possibility to load the Bridge only, completely decoupled from initializing the API client.

Comment (fgiuliani) [reply lib/index.ts:88]: @manuelschroederdev totally forgot about that 😅 I was mainly asking why are we creating this kind of "alias".

**Review by fgiuliani — APPROVED** [review]

**Review by alexjoverm — APPROVED** [review]

**Pull request merged**